demos/node-classification/node2vec/stellargraph-keras-node2vec-node-classification.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>Similar to the other notebook, this cell is duplicated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/536/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I have deleted the extra cell.